samples/export-requirements.txt

```diff
@@ -9,6 +9,6 @@ transformers_stream_generator==0.0.5  # For Qwen
 diffusers==0.34.0 # For image generation pipelines
 timm==1.0.19  # For exporting InternVL2
 torchvision  # For visual language models
-transformers>=4.43 # For Whisper
+transformers>=4.43,<=4.51.3 # For Whisper
```

Comment:
why do we need it?

Reply:
Same here, this is test purpose, since I found that transformers version used for model export are different as python test, would like to regenerate OV cache to check if which version is more stable for CI test.

Reply:
BTW, transformer version should already limited by optimum-intel: [setup.py](https://github.com/huggingface/optimum-intel/blob/55bc71281c4e6317cfbaee46c6fb600d871273dd/setup.py#L32)
```
INSTALL_REQUIRE = [
    "torch>=1.11",
    "optimum@git+https://github.com/huggingface/optimum.git",
    "transformers>=4.36,<4.53",
    "datasets>=1.4.0",
    "setuptools",
    "scipy",
    "onnx",
]
```

Reply:
what is bad with 4.52 transformers?

Reply:
Test failure w/ model export converted with transformer 4.52.4: https://github.com/openvinotoolkit/openvino.genai/actions/runs/16630409967/job/47060373943?pr=2518